src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeaderInfo/ConstraintAccordionViewHeaderInfo.tsx

```diff
@@ -27,43 +29,88 @@ const StyledHeaderText = styled('span')(({ theme }) => ({
     },
 }));
 
+const StyledHeaderWrapper = styled('div')(({ theme }) => ({
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'space-between',
+    borderRadius: '8px',
+}));
+
 interface ConstraintAccordionViewHeaderMetaInfoProps {
     constraint: IConstraint;
     singleValue: boolean;
     expanded: boolean;
     allowExpand: (shouldExpand: boolean) => void;
+    result?: boolean;
+    maxLength?: number;
+    playgroundInput?: PlaygroundRequestSchema;
```

Comment:
Instead of passing playground-specific information to the constraint accordion, could we pass a generic children prop (or something like that) instead? Would be nice if the accordion didn't have to know about the playground.